### PR TITLE
rename namespace to organization

### PR DESCRIPTION
**Pull request by jmbrown412**
This PR removes the usage of `namespace` and replaces it with `organization` in an effort to remove confusion from the application. Also, this was recently done in the `customers` service.

### CODE REVIEW

**Review by vxio — APPROVED** [review]

**Review by darwinz — APPROVED** [review]

**Comment by adamdecaf** [comment]
```
go: goftp.io/server@v0.4.0: reading goftp.io/server/go.mod at revision v0.4.0: unknown revision v0.4.0
make: *** [makefile:10: build] Error 1
```

I changed this recently. Will take a look at why it's failing.

**Comment by jmbrown412** [comment]
> ```
> go: goftp.io/server@v0.4.0: reading goftp.io/server/go.mod at revision v0.4.0: unknown revision v0.4.0
> make: *** [makefile:10: build] Error 1
> ```
> 
> I changed this recently. Will take a look at why it's failing.

@adamdecaf do you want me to wait until you look into this before I merge or can I merge now?

**Comment by adamdecaf** [comment]
https://github.com/moov-io/paygate/pull/597 should fix the build

**Review by darwinz — APPROVED** [review]

**Pull request merged**
